app/controllers/proposal/collection_draft_proposals_controller.rb

```diff
@@ -217,6 +221,15 @@ def user_from_resource
       { name: "#{proposal_urs_user['first_name']} #{proposal_urs_user['last_name']}", email: proposal_urs_user['email_address'] } unless proposal_urs_user.blank?
     end
 
+    def get_approver_emails(approvers_urs)
+      approver_urs_users = retrieve_urs_users(approvers_urs)
```

Comment:
since you are using an env var, you can just use it here

Reply:
done